### PR TITLE
Add .pre-commit-config.yaml file

### PR DESCRIPTION
**Pull request by zsinx6**
Simple pre-commit config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# template-omnivector
\ No newline at end of file
+# template-omnivector
+
+Basic model for use in Omnivector projects.
+
+
+## pre-commit
+To run the pre-commit hooks without installing:
```
Comment (jamesbeedy) [inline README.md:7]: Can we show an example of the installing workflow also?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.0.1
+  hooks:
+    - id: trailing-whitespace
+      args: [--markdown-linebreak-ext=md]
+    - id: check-merge-conflict
+    - id: no-commit-to-branch
+
+- repo: https://gitlab.com/pycqa/flake8
```
Comment (jamesbeedy) [inline .pre-commit-config.yaml:10]: I like this for the example ... would we realistically want to have both black and flake8 in use in the same project though? Possibly there is a use case for this.

**Review by matheushent — APPROVED** [review]
I see no problems, LGTM

File: .pre-commit-config.yaml (hunk 2 above)
Comment (zsinx6) [reply .pre-commit-config.yaml:10]: This is just an example to show both

**Pull request merged**